### ISSUE TITLE
import method name is a reserved keyword

### ISSUE THREAD

**Issue by lgaudez** [cornerstonejs/cornerstoneTools#940]
### Prerequisites

* [ ] Are you running the latest version? -> 3.9.4
* [ ] Are you reporting to the correct repository? 
* [ ] Did you perform a cursory search?

For more information, see the `CONTRIBUTING` guide.

### Description

I'm currently integrating cornerstone tools in a typescript project and I face a problem declaring function "import". Is it possible to rename it with another method name which is not a reserved key word ? 

Thanks

**Comment by dannyrb** [comment]
I've come across this, and it does stink. I find myself doing:

```
// If Babel transpiles this reasonably
import { import as csTools } from 'cornerstone-tools'

const someLibraryFunc = csTools('drawing/someDrawingFunc');

// Or...
import cornerstoneTools from 'cornerstone-tools'

const someLibraryFunc = cornerstoneTools.import('drawing/someDrawingFunc');
```

If Typescript is yelling at you for both of the above, we could try adding an aliased export? Unfortunately, we can't completely rename it without introducing a breaking change and a major version bump.

**Comment by lgaudez** [comment]
Thanks for your answer, an alias could be a nice workaround. We could choose a method name like 'get' or 'retrieve' ? What do you think about this naming?

**Comment by dannyrb** [comment]
@JamesAPetts a full rename may be worth considering for v4.

@Igaudez, naming things is hard. Something more indicative of the behavior might be:

`getModule`, `importModule`

Or to better indicate that these are more prone to changing than top level exports:

`getInternalModule`

**Comment by JamesAPetts** [comment]
`getInternalModule` gets my vote. Lets do this for v4, even:

`import { import as csTools } from 'cornerstone-tools'` makes even our examples difficult to follow sometimes. Quilljs doesn't do _everything_ right.

I'll add this to the Kanban board.

**Comment by JamesAPetts** [comment]
We went with `importInternalModule` haha. Merged in `vNext` in #958. I'll close this for now, this can't be changed until 4.0 as its a breaking change.

**Comment by lgaudez** [comment]
`importInternalModule` is fine. Thx

**Comment by dannyrb** [comment]
> We went with importInternalModule haha. Merged in vNext in #958. I'll close this for now, this can't be changed until 4.0 as its a breaking change.

Per @JamesAPetts's comment, you'll see this change when v4 drops. If you can't wait, please feel free to PR it as an alias'd export and I'll merge the change.
